app/src/main/java/com/github/se/wanderpals/ui/screens/dashboard/DashboardFinanceWidget.kt

```diff
@@ -46,11 +47,14 @@ import com.github.se.wanderpals.ui.screens.trip.finance.FinancePieChart
  * @param viewModel The ViewModel for the Dashboard screen.
  * @param onClick The callback function for when the widget is clicked.
  */
+@SuppressLint("DefaultLocale")
```

Comment:
When you use String.format without explicitly specifying a locale, it defaults to the system’s default locale. However, this can lead to unexpected behavior in certain situations. Instead use for example: String.format(Locale.US,...)

Reply:
Thank you for catching this, I didn't know this.